app/layout/containers/entry.ejs

```diff
@@ -0,0 +1,57 @@
+<div class="container mt-4">
+  <div class="jumbotron mb-5 p-3">
+    <h1 class="text-center mb-3"><b>Try onsite-demo</b></h1>
+    <hr class="my-4">
+    <form class="form-horizontal" method="post">
+    <div class="row justify-content-md-center">
+      <div id="div_id_apiKey" class="form-group required">
+      <label for="apiKey" class="control-label col-md-4  requiredField">
```

Comment:
ver de onde é a class requiredField se for usada não tem problema mas se for tua renomear para "required-field"